app/views/jobseekers/accounts/show.html.slim

```diff
@@ -24,6 +24,7 @@
               = t(".summary_list.change")
               span.govuk-visually-hidden
                 = t(".summary_list.password")
+      p = govuk_link_to t(".close_account"), jobseekers_confirm_destroy_account_path
```

Comment:
shouldnt put links inside p tag like this if there is no other text, just have the link?